PWGCF/DataModel/FemtoDerived.h

```diff
@@ -181,34 +182,60 @@ DECLARE_SOA_TABLE(FemtoDreamDebugParticles, "AOD", "FEMTODEBUGPARTS",
 using FemtoDreamDebugParticle = FemtoDreamDebugParticles::iterator;
 
 /// FemtoDreamTrackMC
-namespace femtodreamparticleMC
+namespace femtodreamMCparticle
 {
 /// Distinuishes the different particle origins
 enum ParticleOriginMCTruth {
-  kPrimary,   //! Primary track or V0
-  kDaughter,  //! Particle from a decay
-  kMaterial,  //! Particle from a material
-  kNotPrimary //! Particle from a material
+  kPrimary,           //! Primary track or V0
+  kDaughter,          //! Particle from a decay
+  kMaterial,          //! Particle from a material
+  kNotPrimary,        //! Not primary particles (kept for compatibility reasons with the FullProducer task. will be removed, since we look at "non primaries" more differentially now)
+  kFake,              //! particle, that has NOT the PDG code of the current analysed particle
+  kDaughterLambda,    //! Daughter from a Lambda decay
+  kDaughterSigmaplus, //! Daughter from a Sigma^plus decay
+  kNOriginMCTruthTypes
 };
 
-DECLARE_SOA_INDEX_COLUMN(FemtoDreamParticle, femtoDreamParticle);
+static constexpr std::string_view ParticleOriginMCTruthName[kNOriginMCTruthTypes] = {"_Primary", "_Daughter", "_Material", "_NotPrimary", "_Fake", "_DaughterLambda", "DaughterSigmaPlus"}; //! Naming of the different OriginMCTruth types
+
```

Comment:
This is a cosmetic suggestion 
Use the same layout as for `ParticleOriginMCTruth`
Return after the open curl bracket, one line for each input and a new line for the closing curl bracket

Reply:
It will help the readability

Reply:
ok, done